src/Microsoft.AspNetCore.Server.IISIntegration/IISMiddleware.cs

```diff
@@ -66,6 +74,17 @@ public IISMiddleware(RequestDelegate next, ILoggerFactory loggerFactory, IOption
                 return;
             }
 
+            // Handle shutdown from ANCM
+            if (HttpMethods.IsPost(httpContext.Request.Method) &&
+                string.Equals(ANCMShutdownRequestPath, httpContext.Request.Path.Value, StringComparison.OrdinalIgnoreCase) &&
```

Comment:
The default for path comparison is `OrdinalIgnoreCase` https://github.com/aspnet/HttpAbstractions/blob/dev/src/Microsoft.AspNetCore.Http.Abstractions/PathString.cs#L332.